plugins/querytranslate/translate.go

```diff
@@ -444,8 +449,8 @@ func GetDefaultScript(backend Backend) string {
 }
 
 // global function to transform the RS API query to _msearch equivalent query
-func TranslateQuery(rsQuery RSQuery, userIP string, queryForId *string) (string, []byte, error) {
-	return translateQuery(rsQuery, userIP, queryForId)
+func TranslateQuery(rsQuery RSQuery, userIP string, queryForId *string, preference *string) (string, []byte, error) {
```

Comment:
@bietkul This change will break noss once you merge this commit into `noss`. I would say an equivalent PR in noss to replace all calls to this function would be good to have since a release will include all the commits from `dev` and that would make builds fail.